metagraph/tests/data/taxo_data/taxo_input.fa

```diff
@@ -0,0 +1,11700 @@
+>gi|1234|ref|NC_2.1|
+AAAGGAAGTTCCCCGTCCAGTTTCCCTTGCAAGTCAGTCTTGCGTTTCCATTGTCTCCTC
+AGGCGTTATAATTGTAGTCCACCCCGATTATGCGTTGTGTACCATTGTTTCGAATGCAAA
+CACACCGCAGCTTCACACCAAACTTAGATATCTTTAAAGTATTGCGTGTCGAGGAGCACA
+TCGTAGACGCCCCATGCTACCTTTTATCTCGTGGCCAGACCTGAAGCGTTTCTTATGAAG
```

Comment:
this file is quite large. Can we avoid uploading it to the repo?

Reply:
This test is generated by a birth-death model simulator, so, I can just replace it with a smaller one if you say that it would be better. Currently both `taxo_input.fa` and `taxo_query.fa` are less than 1MB. Last week this test was very useful for debugging - it was exactly one single sequence that was failing and only on mac. Let me know what you think about it

Reply:
One thing to do here would be to generate a bunch of tests, find those non-trivial where it can fail (like the sequence you found), and push to the repo only them.

In other words, take that sequence, maybe add a few more and that should be enough. If you want more, construct some non-trivial examples to check the edge cases.

Reply:
I will postpone a few days changing the test files, but certainly will do this before merging the pull request. Please let me know about any other things that are not ok in the code